packages/frontend/web/components/Outbrain.tsx

```diff
@@ -3,14 +3,47 @@
 import React from 'react';
 import { Container } from '@guardian/guui';
 
-export const shouldDisplayOutbrain = (config: ConfigType): boolean => {
+interface OutbrainSelectors {
+    widget: string;
+    container: string;
+}
+
+const shouldDisplayOutbrain = (config: ConfigType): boolean => {
     return config.switches.outbrainDCRTest;
 };
 
-export const OutbrainWidget: React.FC<{}> = ({}) => {
+type OutbrainSelectorsType = 'outbrain' | 'merchandising' | 'nonCompliant';
+
+const outbrainSelectorsTypeMapping = {
+    outbrain: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+    merchandising: {
+        widget: '.js-container--commercial',
+        container: '.js-outbrain-container',
+    },
+    nonCompliant: {
+        widget: '.js-outbrain',
+        container: '.js-outbrain-container',
+    },
+};
+
+const getOutbrainSelectorsByType = (
+    type: OutbrainSelectorsType,
+): OutbrainSelectors => {
+    return outbrainSelectorsTypeMapping[type];
+};
+
+const getOutbrainSelectors = (): OutbrainSelectors => {
+    return getOutbrainSelectorsByType('outbrain');
+};
+
+const OutbrainWidget: React.FC<{}> = ({}) => {
+    const selectors = getOutbrainSelectors();
     return (
-        <div className="js-outbrain">
-            <div className="js-outbrain-container" />
+        <div className={selectors.widget}>
```

Comment:
will this apply the class `.js-outbrain` which should probably be dot-less?

Reply:
@gtrufitt 
I am using the same conventions as frontend to use the existing code: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/commercial/modules/third-party-tags/outbrain-load.js

Reply:
I think you're using them different though. In this case you're going to end up adding the wrong class to the div I think

Reply:
Um... looks like you are right. Sorry about that. I have made the correction :)